analytical/__init__.py

```diff
@@ -4,6 +4,5 @@
 
 __author__ = "Joost Cassee"
 __email__ = "joost@cassee.net"
-__version__ = "3.0.0"
-__copyright__ = "Copyright (C) 2011-2020 Joost Cassee and contributors"
-__license__ = "MIT"
+__version__ = "3.1.0"
+__copyright__ = "Copyright (C) 2011-2022 Joost Cassee and contributors"
```

Comment:
I think we can just use the first year, see https://hynek.me/til/copyright-years/
```suggestion
__copyright__ = "Copyright (C) 2011 Joost Cassee and contributors"
```